zerver/views/presence.py

```diff
@@ -147,6 +147,33 @@ def update_user_status_backend(
     return json_success(request)
 
 
+@human_users_only
+@typed_endpoint
+def update_user_status_admin(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    *,
+    user_id: PathOnly[Json[int]],
+    away: Json[bool] | None = None,
```

Comment:
I think we can probably skip the `away` parameter, since it's deprecated and the new endpoint should not need to support it.